src/connectors/gcp/pubsub.rs

```diff
@@ -35,7 +35,7 @@ pub(crate) async fn send_message(
         attributes: HashMap::new(),
         message_id: "".into(),
```

Comment:
should this be read from the metadata so users could specify their message id's?

Reply:
Since here we are sending the message, so the message-id will be assigned by the server. When it's published, we display the message-id as a part of the metadata.
According to the docs for `message_id`: 
ID of this message, assigned by the server when the message is published. Guaranteed to be unique within the topic. This value may be read by a subscriber that receives a PubsubMessage via a Pull call or a push delivery. **It must not be populated by the publisher in a Publish call**.

Reply:
Ohh nice :D that's an interesting API choice of google. Perhaps we add a comment so that in half a year we have not forgotten why it is empty?